float64[] boundary_lat
float64[] boundary_lon
---
bool success
